Modules/_elementtree.c

```diff
@@ -3894,6 +3894,34 @@ _elementtree_XMLParser_close_impl(XMLParserObject *self)
     }
 }
 
+/*[clinic input]
+_elementtree.XMLParser.flush
+
+[clinic start generated code]*/
+
+static PyObject *
+_elementtree_XMLParser_flush_impl(XMLParserObject *self)
+/*[clinic end generated code: output=42fdb8795ca24509 input=effbecdb28715949]*/
+{
+    if (!_check_xmlparser(self)) {
+        return NULL;
+    }
+
+    elementtreestate * const st = self->state;
+
+    if (EXPAT(st, SetReparseDeferralEnabled) == NULL) {
+        return NULL;
```

Comment:
It should not just return NULL, but set some exception.

Reply:
@serhiy-storchaka there will be no flushing needed for versions of Expat that do not expose `XML_SetReparseDeferralEnabled`. That's why I made it return here, it was not a cheap shot but intended. With that said: what do you think?

Reply:
Returning NULL means error. An exception should be raised in this case, otherwise it is a violation of the calling protocol. You can return None (use the `Py_RETURN_NONE` macro) if it is not an error.

Reply:
@serhiy-storchaka interesting, I did not know. Thanks for pointing me to `Py_RETURN_NONE`, will use that with next push.